style.css

```diff
@@ -0,0 +1,128 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background-color: #d3d3d3;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    perspective: 2000px;
+    perspective-origin: 50% 50%;
+}
+.cover{
+    position: absolute;
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+    margin: 0 auto;
+    background-image: url(img/cover.jpg);
+    border-radius: 10px;
+    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
```

Comment:
How does this `10px` affect the shadow ?? @murtazshaikh

Reply:
10px is of blur. More the pixel more the blur shadow will.